plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk

```diff
@@ -533,6 +535,17 @@ group nodePlacement {
                 targets parents   
                 requires nodePlacement.strategy == NodePlacementStrategy.NETWORK_SIMPLEX
             }
+            
+            advanced option labelMerging: MergeRange {
+                label "Merging of Node Labels"
+                description
+                    "Merges labels of nodes if the node width allows it. The values of the pair determine the labels 
```

Comment:
Maybe add a line describing what it means to merge labels i.e. multiple labels on multiple lines, merged labels on one line etc.